go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/kubernetes-csi/external-provisioner
 
-go 1.20
+go 1.21.3
```

Comment:
This must be `1.21`, go mod does not allow patch version. See failed unit tests in this PR.

Reply:
Pardon my ignorance, 1.21.3 is valid in 1.21. Still, all CI jobs fail because they uses 1.20.